rdt-open-reader/src/covidIndonesia/res/values/strings.xml

```diff
@@ -29,4 +29,7 @@
         <item>en</item>
         <item>in</item>
     </string-array>
+    <string name="lbl_back">Back</string>
+    <string name="lbl_gender_id">Female . ID: 99358357</string>
```

Comment:
The string that appears under the patient name is dynamically generated, I don't think this is where it's picked from.

Reply:
yes, it's just moved here instead of hardcoded typos

Reply:
Ok, is the dynamic one translated?

Reply:
No, it's not dynamically translated.

Reply:
I think that's one of the issues raised.

Reply:
so how can we translate the dynamic data? which comes directly from DB.

Reply:
I think we've done this before for the register page?

Reply:
fixed.